src/renderers/dom/client/syntheticEvents/SyntheticEvent.js

```diff
@@ -160,6 +160,7 @@ assign(SyntheticEvent.prototype, {
     this.dispatchConfig = null;
     this._targetInst = null;
     this.nativeEvent = null;
+    this.target = null;
```

Comment:
@zpao I think `currentTarget` is already nullifiled because it is in the `EventInterface` property.

* https://github.com/facebook/react/blob/master/src/renderers/dom/client/syntheticEvents/SyntheticEvent.js#L27
* https://github.com/facebook/react/blob/master/src/renderers/dom/client/syntheticEvents/SyntheticEvent.js#L157-L160